Week1/homework/app.js

```diff
@@ -1,11 +1,157 @@
+      language: "Dutch",
+      author: "Paula Hawkins",
+    },
+    "hanna's_verjaardag": {
+      title: "Hanna's verjaardag",
+      language: "Dutch",
+      author: "Gull Akerblom",
+    },
+    "Een_nieuwe_toekomst": {
+      title: "Een nieuwe toekomst",
+      language: "Dutch",
+      author: "Conor Grennan",
+    },
+    "de_geur_van_regen": {
+      title: "De geur van regen",
+      language: "Dutch",
+      author: "Lieneke Dijkzeul",
+    },
+    "detective_orhan": {
+      title: "Detective Orhan",
+      language: "Dutch",
+      author: "Sadik Yemni",
+    },
+    "last_minute": {
+      title: "Last Minute",
+      language: "Dutch",
+      author: "Linda van Rijn",
+    },
+    "de_man_van_de_bloemen": {
+      title: "De man van de bloemen",
+      language: "Dutch",
+      author: "Johan van Caeneghem",
+    },
+    "ren_amina_ren": {
+      title: "Ren Amina ren",
+      language: "Dutch",
+      author: "Annelie Drewsen",
+    },
+    "het_diner": {
+      title: "Het diner",
+      language: "Dutch",
+      author: "Herman Koch",
+    }
+  
+  }
+  //console.log(bookDetails);
+  
+  const bookImg = {
+    anna_frank_haar_leven : "images/Anna Frank.jpeg",
+    meisje_in_de_trein : "images/meisje in de trein.jpg",
+    hannas_verjaardag : "images/Hannas' verjaardag.jpg",
+    Een_nieuwe_toekomst : "images/nieuwe toekomst.jpg",
+    de_geur_van_regen : "images/de geur.jpg",
+    detective_orhan : "images/detective orhan.jpg",
+    last_minute : "images/last minute.jpg",
+    de_man_van_de_bloemen : "images/man van de bloemen.jpg",
+    ren_amina_ren : "images/ren amina.jpg",
+    het_diner : "images/het diner.jpg"
+    };
+ 
+  function bookList () { 
+
+    let bookcontainer = document.getElementById("mybooks");
+    let div = document.createElement("div");
+    bookcontainer.appendChild(div);
+    let li = document.createElement("li");
+    div.appendChild(li);
+    
+    
+
+
+    for (let i = 0; i< bookTitles.length; i++) {
+
+      // let content = document.createTextNode(bookTitles[i]);
+      // li.appendChild(content);
+      let titleList = document.createElement("li")
+      li.appendChild(titleList);
+      let titleContent = document.createTextNode(bookDetails[bookTitles[i]].title);
+      titleList.appendChild(titleContent);
+      let languageList = document.createElement("li")
+      li.appendChild(languageList);
+      let languageContent = document.createTextNode("Language: " + bookDetails[bookTitles[i]].language);
+      languageList.appendChild(languageContent);
+      let authorList = document.createElement("li")
+      li.appendChild(authorList);
+      let authorContent = document.createTextNode("Author: "+ bookDetails[bookTitles[i]].author);
+      authorList.appendChild(authorContent);
+      let img = document.createElement("img");
+      li.appendChild(img);
+      img.src = Object.values(bookImg)[i];
+      
+      
+  
+
+      li.setAttribute("class", "bookList");
```

Comment:
This is nicely laid out, makes reading which style classes go with which part of the book section very clear.  Well done!